docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -26,13 +26,15 @@ The following screenshot displays the default configuration values:
 -   **Complexity is over:** Pull requests or commits are marked not up to standards if the introduced complexity is higher than the set value.
 -   **Duplication is over:** Pull requests or commits are marked not up to standards if the number of clones introduced is higher than the set value.
 -   **Diff coverage is under:** Pull requests are marked not up to standards if the diff coverage of the pull request is lower than the set value or `∅` ([not applicable](../faq/code-analysis/which-metrics-does-codacy-calculate.md#code-coverage)). This rule is only available for pull requests.
--   **Coverage variation is under:** Pull requests or commits are marked not up to standards if they introduce a variation to coverage lower than the set value. The maximum value is 1%.
+-   **Coverage variation is under:** Pull requests or commits are marked not up to standards if they introduce a variation to coverage lower than the set value.
 
-    To ensure that commits and pull requests:
+    !!! tip
+        Set this gate to a value **between -5.00% and -1.00%**. This will ensure that:
```

Comment:
Why "between" -5%? Why not just "lower than -0.1%"?

Reply:
My idea was that also recommending a lower value will help the users understand the actual purpose of the quality gate (instead of just configuring it so it doesn't block the work of the developers), since we can say this:

> Codacy marks as not up to standards pull requests that **drop coverage by x% or more**

However, I do like starting off with a simpler solution and will give this a go. :+1: